tests/runtimes/android/android_app_gradle_tests.py

```diff
@@ -0,0 +1,235 @@
+"""
+Test for specific needs of Android runtime.
+"""
+# pylint: disable=invalid-name
+import os
+
+from core.base_test.tns_test import TnsTest
+from core.utils.device.device import Device, Adb
+from core.utils.device.device_manager import DeviceManager
+from core.utils.file_utils import File, Folder
+from core.utils.wait import Wait
+from core.settings.Settings import Emulators, Android, TEST_RUN_HOME, AppName
+from core.enums.platform_type import Platform
+from data.templates import Template
+from products.nativescript.app import App
+from products.nativescript.tns import Tns
+
+APP_NAME = AppName.DEFAULT
+
+
+class AndroidRuntimeAppGradleTests(TnsTest):
+
+    @classmethod
+    def setUpClass(cls):
+        TnsTest.setUpClass()
+        cls.emulator = DeviceManager.Emulator.ensure_available(Emulators.DEFAULT)
+        Folder.clean('./' + APP_NAME)
+
+    def tearDown(self):
+        Tns.kill()
```

Comment:
I think we call `Tns.kill()` in `TnsTest.tearDown(self)`.
If so we don't need `def tearDown(self):` for this class at all.

Reply:
done